backend/config/__init__.py

```
@@ -9,3 +12,12 @@ def str_to_list(val: str | None) -> list[str]:
     if not val:
         return []
     return list(map(str.strip, val.split(",")))
+
+
+def parse_time_interval(value: str) -> YFinanceTimeInterval:
+    try:
+        return YFinanceTimeInterval(value)
+    except ValueError as e:
+        raise InvalidTimeIntervalException(
+            f"Invalid time interval, valid intervals are: {", ".join([ti.value for ti in YFinanceTimeInterval])}"
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: expecting '}' (config.__init__, line 22)'

(E0001)